docs/aws/audit/sesmonitoring/rules/ses_malware_scanning_enabled.mdx

```diff
@@ -22,6 +22,222 @@ CBP
+</Accordion>
+
+<Accordion title='Using CLI'>
+1. First, you need to install and configure AWS CLI on your local machine. You can do this by following the instructions provided by AWS. Make sure you have the necessary permissions to access the SES resources.
+
+2. Once the AWS CLI is installed and configured, you can use the following command to list all the receipt rules in a given receipt rule set:
+
+   ```
+   aws ses list-receipt-rules --rule-set-name <Your-Rule-Set-Name>
+   ```
+
+   Replace `<Your-Rule-Set-Name>` with the name of your rule set. This command will return a list of all the receipt rules in the specified rule set.
+
+3. For each receipt rule, check if it has an enabled action of type `Lambda` with an invocation type of `Event`. This action is used to trigger a Lambda function for each email received. The Lambda function should be configured to scan the email for malware. You can do this by inspecting the `Actions` field of the receipt rule. Here is an example of how to do this in Python:
+
+   ```python
+   import json
+
+   # Assume `rule` is a dictionary representing a receipt rule
+   actions = rule['Actions']
+   for action in actions:
+       if action['Type'] == 'Lambda' and action['InvocationType'] == 'Event':
+           # This is a Lambda action. Check if it is configured to scan for malware
+           function_arn = action['FunctionArn']
+           print(f'Lambda function {function_arn} is used to scan emails for malware')
+   ```
+
+4. If none of the receipt rules in the rule set have an enabled Lambda action configured to scan for malware, then SES malware scanning is not enabled.
+</Accordion>
+
+<Accordion title='Using Python'>
+To check if Malware Scanning is enabled in Amazon Simple Email Service (SES), you can use the AWS SDK for Python (Boto3). Here are the steps:
+
+1. **Set up AWS SDK for Python (Boto3):**
+   First, you need to install and configure Boto3. You can install it using pip:
+
+   ```
+   pip install boto3
+   ```
+
+   Then, configure your AWS credentials. You can do this by setting the following environment variables:
+
+   ```
+   AWS_ACCESS_KEY_ID = 'your_access_key'
+   AWS_SECRET_ACCESS_KEY = 'your_secret_key'
+   ```
+
+2. **Create a SES client:**
+   You can create a SES client using Boto3 as follows:
+
+   ```python
+   import boto3
+
+   ses = boto3.client('ses')
+   ```
+
+3. **Get the SES receipt rules:**
+   You can get the receipt rules using the `list_receipt_rules` method. This method returns a dictionary that contains information about the receipt rules.
+
+   ```python
+   response = ses.list_receipt_rules(RuleSetName='your_rule_set_name')
+   ```
+
+4. **Check if Malware Scanning is enabled:**
+   You can check if Malware Scanning is enabled by iterating over the receipt rules and checking the `ScanEnabled` attribute.
+
```

Comment:
<b>Secret:</b> Secret Keyword <br/>
                    <b><a href="https://console.cloudanix.com/code?t=secrets" target='_blank'>Click here for more details</a></b>